### DISCUSSION TITLE
How to fill area between two line series

### DISCUSSION THREAD

**Discussion by mohamadbr32** [tradingview/lightweight-charts#656] [Q&A] [answered]
Hello I have been searching through the documentation and I didn't find any way to fill the area between two line series with a color I'm looking for something like the image below:
<img width="407" alt="fill area" src="https://user-images.githubusercontent.com/48124110/102686096-9b139480-41e5-11eb-9891-62bda3ef5ec6.PNG">

It seems possible but I couldn't find a way to implement that please help, thanks in advance

**Comment by timocov** [comment, marked as answer]
There is a feature request for that, see #140.
